src/__tests__/screens/Accessibility.test.tsx

```diff
@@ -0,0 +1,91 @@
+import React from 'react'
+import { screen, act } from '@testing-library/react'
+import userEvent from '@testing-library/user-event'
+import { fireEvent } from '@testing-library/dom'
+import '@testing-library/jest-dom'
+import { renderWithState } from 'src/__helpers__/render'
+import Accessibility from 'src/screens/Accessibility'
+
+test('can change the content font', async () => {
+  const setFont = jest.fn()
+
+  await act(async () => {
+    renderWithState(<Accessibility />, {
+      contentFontFamily: 'Arial',
+      setContentFontFamily: setFont
+    })
```

Comment:
the renderWithState helper is used like this, the `await act` wrapper is not always needed, it's just for this component that triggers a `setFont` during the initial render, other components may not need it